F/FighterBomber/usr/src/FighterBomberHD.s

```diff
@@ -258,8 +258,20 @@ pl_boot:
 patch:
 		move.w	#$4e75,$11280		;Remove RTE
 		move.w	#$4e71,$110c4		;Remove move.w sr,-(a7)
+		; SMC patch

```

Comment:
now that's worth a patchlist instead of raw moves :)

Reply:
Good point, just tried to keep changes minimal. Can fix that tomorrow.